### PR TITLE
bump up go version to 1.16

### PR DESCRIPTION
**Pull request by wy65701436**
Signed-off-by: Wang Yan <wangyan@vmware.com>

### CODE REVIEW

**Review by ninjadq — APPROVED** [review]
lgtm

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -218,7 +218,7 @@ jobs:
       - name: Set up Go 1.15
```
Comment (reasonerjt) [inline .github/workflows/CI.yml:218]: this should also be updated.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -158,9 +158,10 @@ Harbor backend is written in [Go](http://golang.org/). If you don't have a Harbo
 |   1.9    |    1.12.12    |
 |   1.10   |    1.12.12    |
 |   2.0    |    1.13.15    |
-|   2.1    |    1.14.13     |
+|   2.1    |    1.14.13    |
```
Comment (reasonerjt) [inline CONTRIBUTING.md:161]: Not strictly relevant to this PR, but have we mentioned that notary components are compiled using a different version of Go?

Comment (wy65701436) [reply CONTRIBUTING.md:161]: will raise another PR

**Review by reasonerjt — APPROVED** [review]

**Pull request merged**